src/core/help/RbHelpDatabase.cpp

```diff
@@ -1447,6 +1447,89 @@ mymcmc.run(generations=200000))");
 	help_strings[string("dnPoisson")][string("name")] = string(R"(dnPoisson)");
 	help_arrays[string("dnPoisson")][string("see_also")].push_back(string(R"(dnGeom)"));
 	help_strings[string("dnPoisson")][string("title")] = string(R"(Poisson Distribution)");
+	help_arrays[string("dnPseudo")][string("authors")].push_back(string(R"(Benjamin Redelings)"));
+	help_strings[string("dnPseudo")][string("description")] = string(R"(Fake distribution for adding likelihood evidence)");
+	help_strings[string("dnPseudo")][string("details")] = string(R"(Researchers often want to add information about a parameter that
+already has a prior. For example, we might want to enforce that a
+taxon age t is in the range [a, infinity) or [a,b].  It might seem
+natural to specify a "second prior", but this does not work:
+
+  * A uniform distribution on [a,infinity) cannot be normalized
+    to sum to 1.
+  
+  * A uniform distribution on [a,b] can be normalized, but 
+    normalizing it leads to incorrect results if a and b are
+    themselves random.
+
+The correct approach is to treat this extra information as likelihood 
+information.  If we know that t > a, then we can introduce a 
+hypothetical observation F such that
+
+    Pr(F|t, a) = 1 if t > a
+               = 0 otherwise
+
+For example, F might represent a fossil with a stratigraphic range.
+Conceptually, there exists some (unknown and irrelevant) distribution D
+for F such that
+
+    F ~ D(t,a)
+
+and we only want to specify its likelihood function Pr(F|t, a).
+
+In this case, we say that F is "pseudodata"; it is sometimes called
+"virtual evidence" in the literature.  We write this as
+
+    F ~ dnPseudo(t, pdAbove(a))
+    F.clamp( pseudoObservation() )
+
+Here pdAbove(a) specifies the likelihood function.  We call it the
+pseudodata likelihood. It is NOT a probability distribution, because it
+is not normalized to sum to 1.  The likelihood at x for a pseudodata
+likelihood pd can be obtained with pd.lnLikelihood(x).
+
+Pseudodata likelihoods can be combined in several ways:
+
+  * pd1 && pd2: multiplies the likelihoods.
+        For example pdAbove(a) && pdBelow(b) = pdBetween(a,b)
+
+  * pd1 || pd2: adds the likelihoods.
+
+  * pd + d: shifts the likelihood curve right by d.  
+        For example, pdLogNormal(0,1) + 10 requires that x > 10.
+
+  * pd - d: shifts the likelihood curve left by d.
+
+  * d - pd: computes pd(d-x).)");
```

Comment:
This is exceptionally clear and does a great job of explaining when the user might want to use `dnPseudo()`, and why – thank you!

I only have two minor suggestions:

- When you say that this is known as "virtual evidence" in the statistical literature, it might be worth providing a citation;
- It's better to provide a Markdown file than to edit `RbHelpDatabase` directly – as I'm sure you know, that way we can get the help page to appear both in the RevBayes console and on the website.